CheckingAccount.rb

```diff
@@ -0,0 +1,28 @@
+require './BankAccounts.rb'
+class CheckingAccount < Bank::Account
```

Comment:
Same comment in this class with regards to adding some whitespace between method definitions to provide some additional logical separation